docs/alerts-and-notifications/notifications/README.md

```diff
@@ -1,9 +1,70 @@
 # Notifications
 
-This section includes the documentation of the integrations for both of Netdata's notification methods.
+Netdata supports two ways to send alert notifications: **from Netdata Cloud** or **from the Netdata Agent**.  
+You can use either—or both—depending on how your infrastructure is set up.
 
-<!-- following links are virtual links to a generated page, should not lead somewhere upon click from GitHub -->
+:::tip Need alerts fast?
+Use Cloud for a centralized setup, or Agent for full control on each node.
+:::
 
-- Netdata Cloud provides centralized alert notifications, using the health status data already sent to Netdata Cloud from connected nodes to send alerts to configured integrations. [Supported integrations](/docs/alerts-and-notifications/notifications/centralized-cloud-notifications) include Amazon SNS, Discord, Slack, Splunk, and others.
+---
 
-- The Netdata Agent offers a [wider range of notification options](/docs/alerts-and-notifications/notifications/agent-dispatched-notifications) directly from the Agent itself. You can choose from over a dozen services, including email, Slack, PagerDuty, Twilio, and others, for more granular control over notifications on each node.
+## Notification Methods
+
+### Netdata Cloud (Centralized)
+
+Netdata Cloud collects alert data from all connected nodes and sends notifications through your configured integrations.
+
+→ [See supported Cloud integrations](/docs/alerts-and-notifications/notifications/centralized-cloud-notifications)
+
+**Popular integrations:**
+
+- Amazon SNS
+- Slack
+- Discord
+- Splunk
+- Microsoft Teams
+
+---
+
+### Netdata Agent (Local)
+
+The Agent sends alerts directly from the node, even if it's offline or not connected to the Cloud.
+
+→ [See supported Agent integrations](/docs/alerts-and-notifications/notifications/agent-dispatched-notifications)
```

Comment:
This is 404. I couldn't find a document with a list of Agent notifications. @Ancairon do we have it?